docs/content/foundations/layout-positioning/float.mdx

```diff
@@ -13,6 +13,28 @@ utility_modules:
     utility_class_display_component: 'TestUtilityClassTable'
     utility_class_display_params:
       classes: []
+    utility_examples:
+      - heading:
+        description: |
+        preview: true
+        code: |
+          <section className="utility-examples">
+            <div className="clearfix">
+              <p className="float-none bg-base-lighter padding-2">
+                <span className="site-utility-examples__class">.float-none</span>
```

Comment:
are you able to get rid of all of the uswds site classes `site-utility-examples__class`